src/Suave/Logging/TraceHeader.fs

```diff
@@ -9,45 +9,38 @@ open Suave.Utils.RandomExtensions
 /// of possible values, so you can be fairly certain a given request
 /// id is unique, given a good random number generator.
 type TraceHeader =
-  { /// if this is the 'first' traced request, then trace_id equals
+  { /// If this is the 'first' traced request, then trace_id equals
     /// req_id. If it's the second, then trace_id = req_parent_id
     /// or otherwise third or later then trace_id, req_id and req_parent_id
     /// are all disjunct
-    trace_id      : uint64
-    /// the request id assigned when suave received the http request
+    traceId      : uint64
+
+    /// The request id assigned when suave received the http request
     /// In ZipKin/Dapper-speak, this is the span id
-    req_id        : uint64
+    reqId        : uint64
+
     /// possibly a parent
     /// In ZipKin/Dapper-speak, this is the span parent id
-    req_parent_id : uint64 option }
+    reqParentId : uint64 option }
+
+  static member traceId_     = Property (fun x -> x.traceId) (fun v x -> { x with traceId = v })
```

Comment:
Cool @kolektiv - thanks for that info.  Could you add to this to https://fslang.uservoice.com/forums/245727-f-language/suggestions/6906132-implement-first-class-lensing-lenses-in-f? Thanks